### DISCUSSION TITLE
Change no reply email

### DISCUSSION THREAD

**Discussion by a-mian-lateral** [chamilo/chamilo-lms#5219] [General]
Hello, I need to change the no reply email (that I did not configured during installation) but I can't find any settings that allows me to do it. I checked the web interface and the configuration.php file. Any help please?

EDIT
I just found the option in the web interface, I was searching for "no reply" and found nothing, I should have searched "noreply"

**Comment by ywarnier** [comment]
Great, seems like you found it by yourself. There is also the app/config/mail.conf.php which allows you to set a few other things around e-mails.

**Reply by a-mian-lateral** [reply]
That's the file I modified at the beginning, but I realized that the sender email configuration is a bit tricky to understand, the mail.config file has this comment

//See the function __construct() in main/inc/lib/notification.lib.php for more details

I was looking for the noreply option because seems like the sender email is always the superadmin email.
